refaster-test-support/src/test/java/tech/picnic/errorprone/refaster/test/MatchInWrongMethodRules.java

```diff
@@ -2,8 +2,10 @@
 
 import com.google.errorprone.refaster.annotation.AfterTemplate;
 import com.google.errorprone.refaster.annotation.BeforeTemplate;
+import tech.picnic.errorprone.refaster.annotation.OnlineDocumentation;
 
 /** Refaster rule collection to validate reporting of a match occurring in an unexpected place. */
+@OnlineDocumentation
```

Comment:
Are we expecting this to be documented on the website? Otherwise we should remove this annotation, I think.

Reply:
Nice catch! Will drop.

Reply:
That said, the script should anyway restrict itself to `*/main/*`, since `refaster-runner/src/test/java/tech/picnic/errorprone/refaster/runner/FooRules.java` does need the documentation, but should be ignored regardless :)